plumbing/format/config/common.go

```diff
@@ -5,26 +5,32 @@ func New() *Config {
 	return &Config{}
 }
 
+// Config contains all the sections, comments and includes from a config file
 type Config struct {
 	Comment  *Comment
 	Sections Sections
 	Includes Includes
 }
 
+// Includes is a list of Includes in a config file
 type Includes []*Include
 
-// A reference to an included configuration.
+// Include is reference to an included config file
 type Include struct {
 	Path   string
 	Config *Config
 }
 
+// Comment string with out the prefix '#' or ';'
 type Comment string
 
 const (
+	// NoSubsection token used to avoid provide a section to the Config.Section
```

Comment:
Alternative wording: `NoSubsection token is passed to Config.Section and Config.SetSection to represent the absence of a section`.